packages/hoverable/src/with-hover.spec.tsx

```diff
@@ -98,4 +99,79 @@ describe('withHover HOC', () => {
             expect(nestedElement).not.toHaveStyle(nestedHoverStyle);
         });
     });
+    describe('Children props', () => {
+        const nestedTestColor = 'black';
+
+        it('Should nested component have isActive prop from the root element, if children has hoverColor Prop', () => {
```

Comment:
Why do you need such test?
What is a useCase of such behavior?
I mean, the main idea is that if nested component isn't wrapped by `withHover` HOC, then it shouldn't get any props from its hoverable parent

@vitalyiegorov you're the author of this component, can you put some light here please?